cmd/client/command/visitor.go

```diff
@@ -88,6 +87,12 @@ func (v *StreamVisitor) Visit(fn VisitorFunc) {
 				ExitWithErrorf("error parsing %s: %v", d.doc, err)
 			}
 		}
+		if s.Name == "" {
+			ExitWithErrorf("name cannot be empty: %s", d.doc)
```

Comment:
Error message better be positive expression like `name is empty` instead of `name can not be empty`.

Reply:
updated